simpledotorg/simple-server
# Cphc pilot script

**Story card:** [sc-8988](https://app.shortcut.com/simpledotorg/story/8988/migrate-data-for-the-facilities-in-the-pilot-list)

## This addresses

Adds a script to migrate data to CPHC

## Test instructions

- bundle exec rake dell_demo:migration_demo[<facility_id>]
- Log in to https://ncd-staging.nhp.gov.in/
- You can verify the migrated data using the CPHC UI
